tests/e2e/network/ensureloadbalancer.go

```diff
@@ -144,6 +145,12 @@ var _ = FDescribe("Ensure LoadBalancer", func() {
 		}()
 		By("Waiting for exposure of internal service with specific IP")
 		err = utils.WaitUpdateServiceExposure(cs, ns.Name, serviceName, ip1, true)
+		list, errList := cs.CoreV1().Events(ns.Name).List(metav1.ListOptions{})
+		Expect(errList).NotTo(HaveOccurred())
+		utils.Logf("Events list:")
+		for i, event := range list.Items {
+			utils.Logf(fmt.Sprintf("%d. %v", i, event))
```

Comment:
```suggestion
			utils.Logf("%d. %v", i, event)
```

Reply:
Seems `fmt.Sprintf` is redundant. I'll amend it. Thanks.

Reply:
make sense. please remove the fmt